dftimewolf/lib/telemetry.py

```diff
@@ -0,0 +1,150 @@
+"""Telemetry module."""
+import datetime
+from dataclasses import dataclass
+from typing import Dict, Any, List, Union
+import uuid
+
+# mypy complains when doing from google.cloud import spanner
+try:
+  from google.cloud import spanner  # type: ignore
+  HAS_SPANNER = True
+except ImportError:
+  HAS_SPANNER = False
+
+from dftimewolf import config
+
+
+@dataclass
+class TelemetryEntry:
```

Comment:
Can I suggest a rename? `TelemetryEntry` to me sounds like a single data point, but this is a collection of data points. Maybe `TelemetrySet` or `TelemetryCollection`? Maybe even just `TelemetryDict`